src/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -164,9 +164,9 @@ export class OfferRoute extends Component<OfferProps, OfferState> {
                     defaultValue={null}
                     onChange={ev =>
                       this.setState({
-                        offerValue: Math.floor(
-                          Number(ev.currentTarget.value || "0")
-                        ),
+                        offerValue: +Number(
```

Comment:
Is `+` needed since things are already being coerced with `Number()`?

Reply:
Yes, because `toFixed` converts numbers to strings. It is a little weird, though, to convert to a string to a number to a string to a number... 🤔

Reply:
Just updated this to use `Math.round` instead, which I think is clearer.